api/nvidia.com/resource/gpu/v1alpha1/gpuconfig.go

```diff
@@ -45,11 +46,26 @@ func DefaultGpuConfig() *GpuConfig {
 				Interval: ptr.To(DefaultTimeSlice),
 			},
 		},
+		DriverConfig: &GpuDriverConfig{
+			Driver: NvidiaDriver,
+		},
 	}
 }
 
 // Normalize updates a GpuConfig config with implied default values based on other settings.
 func (c *GpuConfig) Normalize() error {
+	if c.DriverConfig == nil {
+		c.DriverConfig = DefaultGpuDriverConfig()
+	} else {
+		if err := c.DriverConfig.Normalize(); err != nil {
+			return err
+		}
+	}
```

Comment:
Could we just unconditionally call `c.DriverConfig.Normalize` here? Would we be able to initialize a `nil` value?